### ISSUE TITLE
please add global hotkey support

### ISSUE THREAD

**Issue by zecman** [hajimehoshi/ebiten#874]
dear developers of the fantastic ebiten,

thank you very much for your work and the easy to use GUI creation package.

could you please add global hotkey support?

i would like to react to hotkeys even when the ebiten window does not have focus.

**Comment by hajimehoshi** [comment]
Hi,

I'm not sure what is 'hotkey'.

**Comment by hajimehoshi** [comment]
> i would like to react to hotkeys even when the ebiten window does not have focus.

Hmm, I think this is not Ebiten's responsibility but OS's responsibility.

**Comment by zecman** [comment]
please see https://github.com/glfw/glfw/issues/1204, it is explained better there

**Comment by hajimehoshi** [comment]
OK so let's revisit after GLFW implements this since Ebiten depends on GLFW.

Thank you for using Ebiten!

**Comment by zecman** [comment]
Ebiten is a great way to have a gui and everything else. All the thanks to you. I just consume. :-)
Here I found the way to register global hotkeys for windows: https://play.golang.org/p/osbinz7DmN
I will use this way for now since I only target windows atm.

**Comment by zecman** [comment]
@hajimehoshi please take a look at this: https://github.com/robotn/gohook
this exactly does what i need. a bit over-powered (a complete eventlogger for inputs) but already for three platforms. 

i understood, you said you will take a look at it when GLFW implements something...

maybe someone else is hyped by ebiten and needs global inputs before that point of time. ;-)

**Comment by hajimehoshi** [comment]
So you can use this without waiting for Ebiten change?

**Comment by zecman** [comment]
Yes, did some testing yesterday on Win10 - no issues. tested today on Win7 in a very strict network and was very suprised that none of the security systems prevented this global hook... Not even local admin rights were needed. 
Virustotal and Hybrid-Analysis also do not rate this way of realizing "global hotkeys" as suspicious. 

Long answer shot: **Yes**.
